src/apps/OfficeView.js

```diff
@@ -32,26 +32,36 @@ export default function () {
     };
 
     return (
-        <Card>
-            <h1>{office.Name}</h1>
-            <ul>
-                {office.Users.map((user) => (
-                    <li key={user.user.id}>
-                        <Link to={`/sessions/new-with-user/${user.user.id}`}>
-                            {user.name}
-                        </Link>
-                    </li>
-                ))}
-            </ul>
-            <Button to="/" className="lt-button lt-hover" onClick={signout}>
-                Signout
-            </Button>
-            <Button
-                to="/settings"
-                className="lt-button lt-hover"
-                onClick={() => history.push("/settings")}>
-                Settings
-            </Button>
+        <Card className="lt-card lt-shadow">
+            <div class="Office">
+                <p className="lt-card lt-hover text-align:left">
+                    <b>Office Name{office.Name}</b>
+                    <Button className="lt-button lt-hover  --lt-colours-lightgray: #f0f0f0">
+                        Leave
+                    </Button>
+                </p>
+            </div>
+
+            <div class="Desk">
+                <p className="lt-card lt-hover text-align:left">
+                    <ul>
+                        {office.Users.map((user) => (
```

Comment:
This code is creating a list of all the users and a link to join their session. We want to use the interface that you created with Nick's sketches! That means instead of a `li` you'd have a similar structure to your first commit in #29

Reply:
We want an interface which shows the original name and `pull up` button should have the link to join their session right? 
 But how do we access the user names, like if I want 1st users name to get displayed how do I do smth similar to user.name[0].

Reply:
`office.Users` is an array of all the users. When we call `.map` on it, we are transforming each user into a React element so that we can render it. As such, all of the fields in the `office.Users` in Firebase are available here, such as `user.name`, and you can consider how to create a React element for a single user inside the `map` function.

Reply:
Understood!
Another question, we have to do smth similar for dashboard as well? 
I couldn't see how it looks.

Reply:
We should be safe leaving the dashboard as it is for now since you cannot see it anyways.

Reply:
Okay